nixos/modules/services/misc/recyclarr.nix

```diff
@@ -0,0 +1,141 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.recyclarr;
+in
+{
+  options.services.recyclarr = {
+    enable = mkEnableOption "recyclarr service";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.recyclarr;
+      defaultText = literalExpression "pkgs.recyclarr";
+      description = "The recyclarr package to use.";
+    };
+
+    command = mkOption {
+      type = types.str;
+      default = "sync";
+      description = "The recyclarr command to run (e.g., sync).";
+    };
+
+    configFile = mkOption {
+      type = types.either types.path types.str;
+      description = lib.mdDoc ''
+        Path to a compatible recyclarr configuration file.
+        The file should be a valid YAML configuration following the recyclarr specification.
+        For detailed configuration options and examples, see the
+        [official configuration reference](https://recyclarr.dev/wiki/yaml/config-reference/).
+      '';
+    };
+
+    workingDir = mkOption {
+      type = types.path;
+      default = "/var/lib/recyclarr";
+      description = "Working directory for recyclarr.";
+    };
+
+    schedule = mkOption {
+      type = types.str;
+      default = "*-*-* 04:00:00"; # Every day at 4 AM
+      description = "When to run recyclarr in systemd calendar format.";
+    };
+
+    user = mkOption {
+      type = types.str;
+      default = "recyclarr";
+      description = "User account under which recyclarr runs.";
+    };
+
+    group = mkOption {
+      type = types.str;
+      default = "recyclarr";
+      description = "Group under which recyclarr runs.";
+    };
+  };
+
+  config = mkIf cfg.enable {
+
+    users.users = mkIf (cfg.user == "recyclarr") {
+      recyclarr = {
+        isSystemUser = true;
+        description = "recyclarr user";
+        home = cfg.workingDir;
+        group = cfg.group;
+      };
+    };
+
+    users.groups = mkIf (cfg.group == "recyclarr") {
+      ${cfg.group} = { };
+    };
+
+    systemd.tmpfiles.rules = [
+      "d ${cfg.workingDir} 0750 ${cfg.user} ${cfg.group} -"
+    ];
+
+    systemd.services.recyclarr = {
+      description = "Recyclarr Service";
+      after = [ "network.target" ];
+      path = [ cfg.package ];
```

Comment:
Is this needed?

Reply:
No, it was remnant before using `lib.getExe`.